music21/layout.py

```diff
@@ -442,6 +442,19 @@ def __init__(self,
         # in musicxml
         self._barTogether = barTogether
 
+    def fillIntermediateSpannedElements(
+        self,
+        searchStream,  # yikes
+        *,
+        includeEndBoundary: bool = False,
+        mustFinishInSpan: bool = False,
+        mustBeginInSpan: bool = True,
+        includeElementsThatEndAtStart: bool = False
+    ):
+        # There is no fill of intermediate elements in a StaffGroup.  All elements must be
+        # chosen carefully by the client.
+        return
```

Comment:
why?  Why can't it fill in [stream.Part] elements?

Reply:
I guess it could, going by order of Parts in the Score.  But in any case, I have no implementation for this yet, so I'm leaving StaffGroup in the (new) default state, which is that `fill()` does nothing because `self.fillElementTypes` is left empty.